app/ide-desktop/lib/dashboard/src/data/dataLinkSchema.json

```diff
@@ -220,6 +235,124 @@
         "subType": { "title": "Type", "const": "json", "type": "string" }
       },
       "required": ["type", "subType"]
+    },
+    "ExcelFormat": {
+      "title": "Excel",
+      "type": "object",
+      "properties": {
+        "type": { "title": "Type", "const": "format", "type": "string" },
+        "subType": { "title": "Type", "const": "excel", "type": "string" },
+        "section": {
+          "title": "Section",
+          "anyOf": [
+            { "$ref": "#/$defs/ExcelSectionWorkbook" },
+            { "$ref": "#/$defs/ExcelSectionSheet" },
+            { "$ref": "#/$defs/ExcelSectionRange" }
+          ]
+        }
+      },
+      "required": ["type", "subType", "section"]
+    },
+
+    "ExcelSectionWorkbook": {
+      "title": "Workbook",
+      "type": "object",
+      "properties": {
+        "type": { "const": "workbook", "type": "string" },
+        "defaultSheet": {
+          "title": "Default sheet",
```

Comment:
`New Sheet Name`? 
We could change on both sides.

Reply:
I don't think it's clearer.

`Default sheet (for writing)` would be most clear IMO, as it says that it is the _default name_ used _when writing to the workbook_.

Without knowing the context "New sheet name" doesn't tell me much more than "default", other than it _is_ a bit clearer that this is writing-only.

Maybe we should add a `ℹ️` icon at the end, to hint that the user should hover over this to see the `description` which I think provides enough context?